layout/sumu.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+function getSumuLayout(System $system, bool $enable_mobile_layout): Layout {
```

Comment:
Is the HTML structure for this different from New Geisha? I think we should not be duplicating all this if possible - Could we re-use New Geisha's HTML but pass different CSS links in?